index.html

```diff
@@ -218,6 +237,400 @@ <h2>
+        </p>
+        <p>
+          How the [=home tab context=] is presented is at the discretion of the
+          user agent, but it SHOULD have a different appearance to normal tabs.
+        </p>
+        <p>
+          A [=URL=] |url:URL| is said to be <dfn>within home tab scope</dfn> if
+          and only if:
+        </p>
+        <ul>
+          <li>the application [=has a home tab=], and
+          </li>
+          <li>|url| is [=manifest/within scope=] of the manifest, and
+          </li>
+          <li>at least one of the following:
+            <ul>
+              <li>|url| [=URL/equals=] the [=start URL=], with
+              [=URL/equals/exclude fragments=] set to true, or
+              </li>
+              <li>Applying [=URL pattern/match=] given any element of the
+              [=scope_patterns=] member of the [=home_tab=] member of the
+              [=tab_strip=] member of the [=Document/processed manifest=] and
+              |url| returns a {{URLPatternResult}}.
+              </li>
+            </ul>
+          </li>
+        </ul>
+        <p>
+          A URL is is <dfn>outside of home tab scope</dfn> if it is not
+          [=within home tab scope=].
+        </p>
+        <aside class="note">
+          <p>
+            The "home tab scope" (i.e., the set of [=URLs=] which are [=within
+            home tab scope=]) is the set of URLs to which the [=home tab
+            context=] can be navigated . All navigations within the application
+            to a URL [=within home tab scope=] will be performed in the [=home
+            tab context=], and all navigations to a URL [=outside of home tab
+            scope=] will be performed in a different [=application context=]
+            (i.e., another tab), creating one if necessary. The application's
+            [=start URL=] is always part of the home tab scope.
+          </p>
+          <p>
+            For the purpose of matching against the [=start URL=], the
+            [=URL/fragment=] is ignored, but the [=URL/query=] must match
+            exactly.
+          </p>
+          <p>
+            If the application does not [=has a home tab|have a home tab=],
+            then the home tab scope is the empty set.
+          </p>
+        </aside>
+        <section>
+          <h3>
+            Every window has a home tab
+          </h3>
+          <p>
+            If the application [=has a home tab=], whenever a new application
+            window is created (for example when launching the application, or
+            when moving a tab to a new window), the user agent MUST create a
+            new [=home tab context=] in that window. A newly created [=home tab
+            context=] SHOULD be navigated to the [=start URL=], which by
+            definition is [=within home tab scope=].
+          </p>
+        </section>
+        <section>
+          <h3>
+            Navigations concerning the home tab scope
```

Comment:
As per my exploration for link capturing, this has potential to break sites. I recommend linking the 'navigating' keyword here into somewhere in the html spec. Likely in 

Specifically, I recommend probably saying something along the lines of "When [creating a new top-level browsing context and document](https://html.spec.whatwg.org/multipage/document-sequences.html#creating-a-new-top-level-browsing-context),  or [creating a new auxilary browsing context and document](https://html.spec.whatwg.org/multipage/document-sequences.html#creating-a-new-auxiliary-browsing-context), choose the 'home' tab's browsing context when [creating a new browsing context](https://html.spec.whatwg.org/multipage/document-sequences.html#creating-a-new-browsing-context)."

Or -perhaps you'll want to say something like "create a new browsing context that replaces the home tab browsing context".

Example of where this can cause error - if you have named frames, and you open a non-home-tab url (foo.com/away, named frame "B"), and then a user clicks a link that navigates the "B" frame to foo.com, which is in the home tab context.

Or - if the user clicks on a navigate-self link in a tab that is not the home tab, but navigates into the home tab context, what do you want?

It would be really helpful to reference the spec here around navigations, and this helps prevent things like breakages with window.opener and window.frames - the way this is implemented it has to be done where the browsing context gets chosen or created for the navigation in order to not break that.

Reply:
If you can, I think it would be best to hook this into the navigation algorithm. I think that should be possible - I think the correct behavior here is to say that when a navigation occurs within an application context that is 'tabbed', if that navigation [creates a new browsing context](https://html.spec.whatwg.org/multipage/document-sequences.html#creating-a-new-browsing-context), then it should use the 'home tab' browsing context.